v3/util/qc_stmt.go

```diff
@@ -1,5 +1,5 @@
 /*
- * ZLint Copyright 2024 Regents of the University of Michigan
+ * ZLint Copyright 2025 Regents of the University of Michigan
```

Comment:
It took me a bit to understand the changes in this file, but from what I can see it's simply lifting out complex blocks into their own functions.

Is there anything that you would like to call out for closer review? It's difficult to tell what is new and what is simply re-arranging code for readability.

Reply:
There were some older PRs that didn't make it to the main branch. I believe this version was used in those PRs and should include all the necessary functionality related to QC. So it is a mix of extracting existing functionality and adding features that are not currently required. Since the other ETSI lint tests are running fine, I do not think it is necessary to take a closer look.